config/sync/core.entity_form_display.paragraph.card_grid_automatic.default.yml

```diff
@@ -34,6 +36,14 @@ targetEntityType: paragraph
 bundle: card_grid_automatic
 mode: default
 content:
+  field_card_grid_link:
```

Comment:
Please make the field name more generic (or use an existing one)

e.g. `field_link` / `field_more_link` or similar.

That way, we can re-use logic in the future.